### ISSUE TITLE
Database version 32.12 is not supported for ICompanyAdminBusiness

### ISSUE THREAD

**Issue by baljeetbhardwaj** [NCRCounterpointAPI/APIGuide#106]
Hi,

when I am trying filter users into edit company admin from API admin console, I am getting following error

"Database version 32.12 is not supported for ICompanyAdminBusiness."

Please help!!

Thanks in advance.

**Comment by baljeetbhardwaj** [comment]
any luck my friends?

**Comment by MikeRitchie** [comment]
Do all company level calls return this result, or just when filtering users?

Also, what version of the API are you running?

**Comment by baljeetbhardwaj** [comment]
Yes, I am able to create Admin User and Delete Admin User. when I am trying to filter users then I got this error message. 

API version  is 8.5.3

Please check attached image
![cp](https://user-images.githubusercontent.com/30104271/38992155-7ae4f304-43fd-11e8-925d-55ab86cd9b3d.JPG)
